docs/content/management-bundles.md

```diff
@@ -105,6 +105,46 @@ in bundle responses to identify the revision of the bundle. OPA will include the
 check the `If-None-Match` header and reply with HTTP `304 Not Modified` if the
 bundle has not changed since the last update.
 
+#### HTTP Long Polling
```

Comment:
Huh that's a surprise. Unrelated to the change, isn't it?

Reply:
Fair point. We've already added the code changes that implement long-polling but never really exposed the feature. Initially the changes for long polling and delta bundles were slated to go close to each other but that did not work out. I've added the long polling explanation here to emphasize that long polling and delta bundle together can provide good performance improvements. If you think this should be a separate PR, I'm open to that.

Reply:
Nah it's ok. Thanks for the rationale